src/EntityFramework.Commands/tools/EntityFramework.psm1

```
@@ -426,6 +426,63 @@ function Scaffold-DbContext {
     ShowConsole
 }
 
+# 
+# Scaffold-Directives
+#
+
+Register-TabExpansion Scaffold-Directives @{
+    Project = { GetProjects }
+    StartupProject = { GetProjects }
+}
+
+
+<#
+.SYNOPSIS
+    Scaffolds a runtime directive file for .NET Native.
+
+.DESCRIPTION
+    Scaffolds a runtime directive file for .NET Native. 
+	These directives instruct the compiler to include metadata for types needed at runtime.
+
+.PARAMETER Project
+    Specifies the project to use. If omitted, the default project is used.
+
+.PARAMETER StartupProject
+    Specifies the startup project to use. If omitted, the solution's startup project is used.
+
+.PARAMETER Environment
+    Specifies the environment to use. If omitted, "Development" is used.
+
+.LINK
+    about_EntityFramework
+#>
+
+function Scaffold-Directives {
+    [CmdletBinding(PositionalBinding = $false)]
+    param (
+        [string] $Project,
+        [string] $StartupProject,
+        [string] $Environment)
+
+	$values = ProcessCommonParameters $StartupProject $Project
+    $dteStartupProject = $values.StartupProject
+	$config = $dteStartupProject.ConfigurationManager.ActiveConfiguration.ConfigurationName
+
+	if ( $config -ne "Debug") {
+		Write-Warning "Cannot run directive scaffolding in '$config' mode. Set the solution to 'Debug' and try again."
```

Also, this is terminating so it should `throw` instead of warn.